### PR TITLE
Py idiomatic API: Doc sets, datatypes, FP

### PR DESCRIPTION
**Pull request by alex-ozdemir**

### CODE REVIEW

**Comment by alex-ozdemir** [comment]
Not yet ready for review.

**Comment by alex-ozdemir** [comment]
Now, it is ready.

File: docs/api/python/z3compat/fp.rst [hunk 1]
```diff
@@ -1,2 +1,126 @@
...
+
+
+Testers
+-------------------
+.. autofunction:: cvc5_z3py_compat.is_fp_sort
+.. autofunction:: cvc5_z3py_compat.is_fprm_sort
+.. autofunction:: cvc5_z3py_compat.is_fprm
+.. autofunction:: cvc5_z3py_compat.is_fprm_value
+.. autofunction:: cvc5_z3py_compat.is_fp
+.. autofunction:: cvc5_z3py_compat.is_fp_value
```
Comment (nafur) [inline docs/api/python/z3compat/fp.rst:89]:
```suggestion
.. autofunction:: cvc5_z3py_compat.is_fp_sort
.. autofunction:: cvc5_z3py_compat.is_fp
.. autofunction:: cvc5_z3py_compat.is_fp_value
.. autofunction:: cvc5_z3py_compat.is_fprm_sort
.. autofunction:: cvc5_z3py_compat.is_fprm
.. autofunction:: cvc5_z3py_compat.is_fprm_value
```

**Review by nafur — APPROVED** [review]
LGTM. Thanks for the changes!

**Pull request merged**